spec.md

```diff
@@ -134,41 +133,57 @@ the paths to label objects which can be found underneath the group:
 
 Unlisted groups MAY be labels.
 
-### "label""
-
-
-
-### "color"
-
-The `color` key defines an image that is "labeled", i.e. every unique value in the image
-represents a unique, non-overlapping object within the image. The value associated with
-the `color` key is another JSON object in which the key is the pixel value of the image and
-the value is an RGBA color (4 byte, `0-255` per channel) for representing the object:
-
-```
-{
-  "color": {
-    "1": 8388736,
-    ...
-```
-### "image"
-
-The `image` key is an optional dictionary which contains information on the image the label is associated with.
-If included it must include a key `array` whose value that is either:
-- A relative path to a Zarr image group, for example:
-    ```
-    {
-      "image": {
-        "array": "../../0"
-      }
+### "image-label" metadata
+
+Groups containing the `image-label` dictionary represent an image segmentation
+in which each unique pixel value represents a separate segmented object.
```

Comment:
```suggestion
in which each unique pixel value represents a separate segmented object.
Only integer pixel types are supported.
```
Trying to compare a floating-point pixel value with the floating point in the JSON metadata is difficult.